proto/provenance/metadata/v1/tx.proto

```diff
@@ -73,7 +80,7 @@ message MsgMemorializeContractRequest {
 // MsgMemorializeContractResponse from a contract execution memorialization request
 message MsgMemorializeContractResponse {}
 
-// MsgChangeOwnershipRequest records the result of a change ownership p8e contract execution
+// MsgChangeOwnershipRequest records the result of a change ownership contract execution
```

Comment:
This function is a P8e only  concept ... it should remain tagged as P8e (or more likely) removed entirely